src/lib389/lib389/idm/group.py

```diff
@@ -112,6 +112,23 @@ def __init__(self, instance, basedn, rdn=DEFAULT_BASEDN_RDN):
         else:
             self._basedn = '{},{}'.format(ensure_str(rdn), ensure_str(basedn))
 
+    def create_test_group(self, gid=2000):
```

Comment:
I do not think this function is used. BTW as it is only used for test it should not be in lib389 but rather in the test